source/includes/_email_survey.md

```diff
@@ -25,6 +25,7 @@ survey_settings | Hash | See **survey_settings** parameters below
 subject | String | Override default subject line (standard NPS/CES/CSAT question used by default). Can handle placeholder values surrounded by {{my_value}}
 intro | String | Intro message for your email. Displayed between logo and question. Default: none. Can handle placeholder values surrounded by {{my_value}}
 context | Hash Array | Array of hashes containing the values to be replaced in subject and/or intro
+delay | Integer | Value (in days)  that specifies how many days will the survey wait until being fired
```

Comment:
There's kind of like an extra space there after the parenthesis. Can you verify with Ray or Dan the language of the description?